src/segment_media_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStringPlayingMedia(t *testing.T) {
+	expected := "\ue602 Candlemass - Spellbreaker"
+	s := &media{}
+	s.info.MediaInfo.Title = "Candlemass"
+	s.info.MediaInfo.Artist = "Spellbreaker"
+	s.info.Playback.PlaybackState = 5
+	assert.Equal(t, expected, s.string())
+}
+
+func TestStringPausedMedia(t *testing.T) {
```

Comment:
We moved to a specific style of tests. See [here](https://github.com/JanDeDobbeleer/oh-my-posh/blob/572149b176b864775edae0e531ed38153a12d66a/src/segment_battery_test.go#L17) for more details.